pkg/network/ebpf/c/http-maps.h

```diff
@@ -12,12 +12,15 @@ BPF_HASH_MAP(http_in_flight, conn_tuple_t, http_transaction_t, 1)
 /* This map used for notifying userspace that a HTTP batch is ready to be consumed */
 BPF_PERF_EVENT_ARRAY_MAP(http_notifications, __u32, 0)
 
-/* This map stores finished HTTP transactions in batches so they can be consumed by userspace*/
-BPF_HASH_MAP(http_batches, http_batch_key_t, http_batch_t, 1024)
+/*
+  This map stores finished HTTP transactions in batches so they can be consumed by userspace
+  Size is set dynamically during runtime and must be equal to CPU*HTTP_BATCH_PAGES
+ */
+BPF_HASH_MAP(http_batches, http_batch_key_t, http_batch_t, 0)
```

Comment:
this map will also be changed to a `PERCPU_ARRAY` in a future PR, but currently userspace needs to read entries from multiple CPUs, which precludes this change.